README.md

```diff
@@ -245,6 +255,17 @@ There are two ways to specify the role: use the built-in role or create a custom
     AZURE_CLIENT_ID=`az ad sp list --display-name "velero" --query '[0].appId' -o tsv`
     ```
 
+2. Assign the service principal roles:
+
+    ```bash
+    # with custom role
+    az role assignment create --role $AZURE_ROLE --assignee "velero" --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
+    # with built-in roles
+    az role assignment create --role "Contributor" --assignee "velero" --scope /subscriptions/$AZURE_SUBSCRIPTION_ID
+    # not needed when subscriptionId and resourceGroup is set (deprecated)
```

Comment:
Same comment with above, let's remove `# not needed when subscriptionId and resourceGroup is set (deprecated)`